packages/round-manager/package.json

```diff
@@ -59,5 +59,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "autoprefixer": "^10.4.7",
+    "postcss": "^8.4.14",
+    "tailwindcss": "^3.0.24"
```

Comment:
I don't think this is a dev-dependency.

Reply:
that's the directive here -> https://tailwindcss.com/docs/guides/create-react-app

+ i did a trial production build, worked since it's compiled static files

Reply:
that's the directive here -> https://tailwindcss.com/docs/guides/create-react-app

[+] i did a trial production build, worked since it's compiled static files